invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/custom_fields/CustomFields.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from "react";
+import { loadCustomFields } from "./utils";
+import { AccordionField } from "react-invenio-forms";
+
+export class CustomFields extends Component {
+  state = { sections: [] };
+
+  async componentDidMount() {
+    // Custom fields
+    loadCustomFields(this.props.config).then((sections) => {
```

Comment:
can we use async/await here too?
```suggestion
    const sections = await loadCustomFields(this.props.config);
```
please add also the error handler with a nice msg for the try/catch